### DISCUSSION TITLE
Catalogue upload failures multiple source databases (analysis id 0 is missing)

### DISCUSSION THREAD

**Discussion by kdenturc** [EHDEN/CatalogueExport#57] [General]
Hi,
We encounter failures uploading our catalogue export.
The CatalogueExport run can't seem to handle the fact that we have multiple source databases.
We need to manipulate id 0 to one database.
(status notification --> failed: analysis id 0 is missing, try (re)running the plugin CatalogueExport on your database)

Anyone else having this issue?

Kind regards,
Kim RADar team AZ Delta Belgium

**Comment by MaximMoinat** [comment]
Hi @kdenturc, the Dashboard Upload in the EHDEN Portal expects the first row of data to be of analysis 0. Apparently this is not the case for you.

Could you explain a bit more how having multiple source databases causes this issue?
